pypasta/LinuxMailCharacteristics.py

```diff
@@ -221,48 +220,35 @@ class LinuxMailCharacteristics:
 
     def _is_from_bot(self, message):
         email = self.mail_from[1].lower()
-        bots = ['broonie@kernel.org', 'lkp@intel.com']
-        potential_bot = True in [bot in email for bot in bots]
-
-        if potential_bot and \
-           email_get_header_normalised(message, 'x-patchwork-hint') == 'ignore':
-            return True
-
+        bots = ['tip-bot2@linutronix.de', 'tipbot@zytor.com', 'noreply@ciplatform.org', 'syzbot',
+                'syzkaller.appspotmail.com']
```

Comment:
Ah, thanks for picking this up! We have another undetected bot left: The Hulk Robot <hulkci@huawei.com>

Reply:
... And I'll move the constants to the top of the class.